### DISCUSSION TITLE
Can you add a parameter of distance to group words.

### DISCUSSION THREAD

**Discussion by vijuc895** [pymupdf/PyMuPDF#1110] [Q&A]
Hi Team,

Is there is any parameter of char_margin to group two word if they are really close.

Example: If have **`Folio No`** in pdf so in extractWord() it is giving as two words **`"Folio"`** and **`"No"`** separately but I want it as single word **`"Folio No"`**.

TIA.

**Comment by JorjMcKie** [comment]
You actually have all the information to add this type of logic yourself, as a programmer:
* if two subsequent words with the same block number and the same line number have bboxes where `bbox1.x1` is close to `bbox2.x0`, then join them.
* you can make this decision also dependent on the bbox heights - which is related to the line height
* you can experiment with the text extraction flag bit `fitz.TEXT_INHIBIT_SPACES`. Set it to off to prevent that MuPDF adds extra spaces when bboxes of two characters in sequence are not adjacent.

**Comment by vijuc895** [comment]
Thanks for the quick response, will try and share you the update on the same, Thanks Again!
